lists-ui/src/components/FiltersSection.tsx

```diff
@@ -180,7 +182,7 @@ export const FacetComponent = memo(
           </Group>
         )}
         { isBooleanFacet && isShowFlagLabel && (
-            <Text size='md' className={classes.facetHeader} style={{ borderTop: '1px solid var(--mantine-color-default-border)' }}>
+            <Text size='md' span className={classes.facetHeader} style={{ width: '100%', paddingBottom: '4px', borderTop: '1px solid var(--mantine-color-default-border)' }}>
```

Comment:
style here overrides facetHeader css, which is also referenced in another block. Might be a better way of doing this...

Reply:
Alternatives that come to mind are; use 2 classes, or replace with a new class, or a variant.